tests/nosetests/pyanaconda_tests/module_security_test.py

```diff
@@ -50,6 +50,14 @@ def setUp(self):
         self.callback = PropertiesChangedCallback()
         self.security_interface.PropertiesChanged.connect(self.callback)
 
+    def _test_dbus_property(self, *args, **kwargs):
```

Comment:
Please use the name `_check_dbus_property()`. The `test` prefix is used for test cases so it's better to use check for helper logic.

Reply:
Fixed.